Dockerfile

```diff
@@ -1,10 +1,14 @@
-FROM php:8.2-alpine
+FROM ghcr.io/phpstan/phpstan:latest-php8.2
```

Comment:
Moet deze versie misschien nog een variable zijn die uit het project gehaald moet worden zodat die bij verschillende projecten ingezet kan worden?

Reply:
is momenteel niet te doen. Je kan alleen args meesturen bij runtime van de container, niet bouwen van image